specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2020-10-19-preview/desktopvirtualization.json

```diff
@@ -0,0 +1,3795 @@
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SessionHostProperties"
+        }
+      }
+    },
+    "SessionHostProperties": {
+      "description": "Schema for SessionHost properties.",
+      "type": "object",
+      "properties": {
+        "lastHeartBeat": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Last heart beat from SessionHost."
+        },
+        "sessions": {
+          "type": "integer",
+          "description": "Number of sessions on SessionHost."
+        },
+        "agentVersion": {
+          "type": "string",
+          "description": "Version of agent on SessionHost."
+        },
+        "allowNewSession": {
+          "type": "boolean",
+          "description": "Allow a new session."
+        },
+        "virtualMachineId": {
+          "type": "string",
+          "description": "Virtual Machine Id of SessionHost's underlying virtual machine.",
+          "readOnly": true
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "Resource Id of SessionHost's underlying virtual machine.",
+          "readOnly": true
+        },
+        "assignedUser": {
+          "type": "string",
+          "description": "User assigned to SessionHost."
+        },
+        "status": {
+          "enum": [
+            "Available",
+            "Unavailable",
+            "Shutdown",
+            "Disconnected",
+            "Upgrading",
+            "UpgradeFailed"
+          ],
+          "x-ms-enum": {
+            "name": "Status",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "Status for a SessionHost."
+        },
+        "statusTimestamp": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The timestamp of the status.",
+          "readOnly": true
+        },
+        "osVersion": {
+          "type": "string",
+          "description": "The version of the OS on the session host."
+        },
+        "sxSStackVersion": {
+          "type": "string",
+          "description": "The version of the side by side stack on the session host."
+        },
+        "updateState": {
+          "enum": [
+            "Initial",
+            "Pending",
+            "Started",
+            "Succeeded",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "UpdateState",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "Update state of a SessionHost."
+        },
+        "lastUpdateTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The timestamp of the last update.",
+          "readOnly": true
+        },
+        "updateErrorMessage": {
+          "type": "string",
+          "description": "The error message."
+        },
+        "sessionHostHealthCheckResult": {
```

Comment:
In the example it looks like this is returning a JSON embedded in the string value. Why isn't it modeled as an object.

Reply:
So I talked to the developer who worked on the property. She said the sessionHostHealthCheckResult property was a string in the DB and decided to surface the value as a string.
Is this not acceptable?

Reply:
removed the property will re add when change is made